tests/snapshot/test_voting.py

```diff
@@ -106,4 +106,4 @@ def test_create_wait_enact(helpers, vote_time, call_target, vote_ids_from_env):
             assert_expected_diffs(
                 step_name, diff, {"votesLength": ValueChanged(from_val=votesLength + 1, to_val=votesLength + 2)}
             )
-        assert_no_diffs(step_name, diff)
+        assert_no_diffs(step_name, diff)
```

Comment:
it looks like an extra space or something like formatting, it's better if there is no diff

Reply:
@zuzueeka There are really no changes here. I accidentally moved something, then put it back the way it was